src/documentation/tooling/index.md

```diff
@@ -0,0 +1,208 @@
+> - (4) Different representation forms  
+
+### Sustainability
+
+The [documentation sustainability](../sustainability/) section establishes that
+
+> [documentation] sustainability must be ensured by choosing sustainable tooling 
+[...].
+
+Software sustainability covers many aspects, including community parameters such
+as size, number and frequency of contributions; whether a code is open source;
+the programming language it is implemented with; ease of installation; its
+dependency tree, etc.
+
+Accordingly, there is currently no canonic definition of software 
+sustainability, as discussed in [1]. One of the definitions given in [1] seems
+operationable enough to use in the context of documentation tooling:
+
+> Sustainable software is software which is:  
+> - Easy to evolve and maintain
+> - Fulfils its intent over time
+> - Survives uncertainty
+> - Supports relevant concerns (Political, Economic, Social, Technical,
+> Legal, Environmental)
+
+[1] D. S. Katz, 'Defining Software Sustainability', 2016. [Online]. 
+Available: <https://web.archive.org/web/20181122140500/https://danielskatzblog.wordpress.com/2016/09/13/defining-software-sustainability/>. 
+[Accessed: 22-Nov-2018]
+
+### Single tool "toolchain"
+
+One of the key motivations behind the choice of tooling is to have *one single 
+software* which we use for all of our (textual) documentation. This makes
+maintainership transitions easier as it requires less learning effort from new
+maintainers, as well as contributors.
+
+### Usability
+
+#### Human-readable sources
+
+In the event of a failure in the documentation *software*, the documentation
+*sources* must be readable and well-structured enough to function as a fallback
+in place of, e.g., HTML- or PDF-rendered documentation.
+
+We use a combination of hierarchical directory structures and human-readable
+source file formats to achieve this.
+
+#### Javadoc integration
+
+As **Hexatomic** is written in Java, we use 
+[Javadoc](https://en.wikipedia.org/wiki/Javadoc) to document source code 
+in situ. This way, we can generate 
+[API](https://en.wikipedia.org/wiki/Application_programming_interface) 
+documentation in HTML directly from the source code via the 
+[javadoc](https://docs.oracle.com/javase/1.5.0/docs/tooldocs/solaris/javadoc.html) 
+tool. This specific API documentation format is used standardly across most
+Java software projects[^ossrh-requirements] and is something that code contributors will expect to
+find.
+
+In order to boost findability of the API documentation, it would be helpful
+to be able to integrate the Javadoc HTML in the text documentation for 
+developers as easily as possible, ideally through native support for this by the
+documentation software.
+
+[^ossrh-requirements]: For example, all release artifacts of open source 
+projects that are hosted on "Maven Central", the main repository for the 
+dominant build system for Java (cf. [2]), are uploaded through the [Open Source
```

Comment:
Footnote formatting should be using the markdown syntax for footnotes.

Reply:
It's not a footnote but an IEEE formatted reference. I found it better to include any references at the bottom of the respective paragraph.